### PR TITLE
chore: Add CODEOWNERS

### PR DESCRIPTION
**Pull request by vadorovsky**
For each subdirectory, require a review from at least one code owner.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+psp-template/ @ananas-block @sergeytimoshin @vadorovsky
+rpc/ @sergeytimoshin @SwenSchaeferjohann
+scripts/ @vadorovsky
+third-party/ @vadorovsky
+tsconfig/ @SwenSchaeferjohann
+utils/ @vadorovsky
+verifier-sdk/ @ananas-block @vadorovsky
+web/ @SwenSchaeferjohann
+xtask/ @vadorovsky
+zk.js/ @ananas-block @SwenSchaeferjohann
```
Comment (sergeytimoshin) [inline CODEOWNERS:20]: add me here please

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,20 @@
+account.rs/ @sergeytimoshin
+assets/ @SwenSchaeferjohann
+circuit-lib/ @ananas-block @sergeytimoshin
+cli/ @ananas-block
```
Comment (sergeytimoshin) [inline CODEOWNERS:4]: add me here please

File: CODEOWNERS (hunk 1 above)
Comment (vadorovsky) [reply CODEOWNERS]: Done

File: CODEOWNERS (hunk 2 above)
Comment (vadorovsky) [reply CODEOWNERS]: Done

**Review by SwenSchaeferjohann — APPROVED** [review]
thx!

**Pull request merged**